### ISSUE TITLE
Compute

### ISSUE THREAD

**Issue by christophschuhmann** [salesforce/BLIP#2]
Your BLIP is AWESOME.

I am one of the chairmen and founders of LAION. ( https://laion.ai )

We have much more data, then you used for BLIP. 

May i ask how much compute you used to train the ViT-L BLIP?
Pre-Training and finetuning?

If you are interested in more image-text-data, let us know. :)

**Comment by LiJunnan1992** [comment]
Thanks for releasing LAION! It is really important for the community to have a publicly-available image-text dataset. 

For ViT-L pre-training, I used 32 A100 GPUs. For fine-tuning I used 16 A100. Gradient checkpointing was used to reduce GPU memory usage. 

I'll let you know if I need more data. Please also let me know if you want to include our synthetic captions as part of LAION.  We found the synthetic captions to be very beneficial for vision-language pre-training.

**Comment by styler00dollar** [comment]
How long did the training process take to pre-train and finetune a model?

**Comment by LiJunnan1992** [comment]
Finetuning takes a few hours for most tasks.

Pre-training a model with ViT-B on 14M images takes 3-4 days. Our best model pre-trained on 125M images takes 8-9 days.

**Comment by YoojLee** [comment]
Hi, I would like to ask you which A100 GPUs you have used for this work! The ones with 40GB VRAMs or 80GB? Thanks!
